estate/models/estate_property.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate module"
```

Comment:
This is the description for a specific model not the entire module, right now it is a little bit confusing